common/src/main/java/com/netflix/conductor/common/metadata/workflow/WorkflowDef.java

```diff
@@ -66,6 +66,9 @@
 	@ProtoField(id = 9)
 	private boolean restartable = true;
 
+	@ProtoField(id = 10)
+	private boolean terminalWorkflowListenerEnabled = false;
```

Comment:
consider naming it workflowStatusListenerEnabled to be consistent with interface name, also be generic as we should also notify on other status changes, e.g. when a workflow is restarted.

Reply:
renamed